docs/reference/overview_of_strategies.rst

```diff
@@ -392,7 +392,7 @@ earned in the previous round. WSLS is also known as "Win-Stay-Lose-Switch" and
 - :math:`P(C\,|\,DC) = 0`
 - :math:`P(C\,|\,DD) = 1`
 
-*TF2T came 7th in average score and 13th in wins in S&P's tournament.*
```

Comment:
NOTE: This is a possible minor typo completely unrelated to this PR.  It didn't seem worth raising a separate issue. (1 of 2)